packages/database/src/DatabaseModel.js

```diff
@@ -201,7 +201,15 @@ export class DatabaseModel {
   }
 
   generateInstance = async (fields = {}) => {
-    const data = await this.getDatabaseSafeData(fields);
```

Comment:
I am 95% sure the `getDatabaseSafeData` method was just called out of convenience here. What we actually want is just an object with the expected records filled out, but on _retrieving_ records we shouldn't need to do the same sanitization as when we're _inserting_ records into the db (and in fact, with the addition of stringifying json arrays, we wouldn't want to)

Reply:
Sounds good!